prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -100,6 +100,41 @@ protected SimpleDateFormat initialValue() {
     private static ConcurrentHashMap<Entity, List<MetricInfo>> entityBacklogs = new ConcurrentHashMap<>();
 
     @Override
+    public void onAdd(Entity entity) throws FalconException{
+        //DO Nothing
```

Comment:
We don't need to handle it here.It's already being handled in EntitySLAService which will create the pending instances and EntitySLAAlertService will alert the BacklogEmitterService in case anyone of them misses the SLA.